api/util/errorLog.js

```diff
@@ -9,13 +9,13 @@ const ErrorLog = require('../models/ErrorLog');
  * @param {string} req.body.apiEndpoint - The location of the error
  * @param {string} req.body.errordescription - The description of the error
  */
-async function addErrorLog(userEmail, errorTime, apiEndpoint, errorDescription){
+async function addErrorLog(req){
```

Comment:
lets call this something better like

```suggestion
async function addErrorLog(errorToAdd){
```